### DISCUSSION TITLE
Outline

### DISCUSSION THREAD

**Discussion by johko** [huggingface/computer-vision-course#80] [07. Video & Video Processing]
Hello everyone,

Together with my collaborators, we've drafted a curriculum for our section. We're keen on getting feedback from the HF team during our writing. Here's what we've got so far:

Proposed Curriculum
INTRODUCTION (introduction-to-video.mdx)

- [ ] What is a Video? – Delving into the Basic Concept (Assigned to @DiwakarBasnet, Estimated: by Nov 25th)
- [ ]  Role of Video Processing in AI and CV (Assigned to @wonhyeongseo, Estimated: by Nov 25th)

VIDEO PROCESSING BASICS (video-processing-basics.mdx)
 
- [ ] Video vs. Image – Exploring Key Processing Differences (Assigned to @DiwakarBasnet, Estimated: by Nov 25th)
- [ ]  Temporal Continuity & Motion Estimation (Assigned to @DiwakarBasnet, Estimated: by Nov 25th)
- [ ]  Key Techniques: Stabilization, Background Subtraction, and Object Tracking

DEEP LEARNING FOR VIDEO (deep-learning-for-video.mdx)

- [ ] Overview of Previous SOTA Models – Including 3D CNNs like (2+1)D Resnets and Two Stream Nets (Assigned to @sazio , Time: To be specified)
- [ ]  Transformers in Video Processing – Highlighting differences in ViT for Image vs. Video (Assigned to @DiwakarBasnet and @sazio , Time: To be specified)
- [ ]  Key Applications: Action Recognition and Video Captioning

PRACTICAL APPLICATIONS & CHALLENGES (applications-and-challenges.mdx)

- [ ]  Practical Applications: Surveillance, Streaming, Autonomous Driving, AR/VR, Healthcare, and Sports Analytics (Note: Might be too much; consider narrowing down for depth, Assigned to @sanae-a11y, Estimated: To be specified)
- [ ]  Challenges: Real-time Constraints & Privacy Concerns (Assigned to @wonhyeongseo, Estimated: by Nov 25th)

HANDS-ON DEMO (Jupyter Notebooks)

- [ ] Simple Video Stabilization
- [ ]  Object Tracking
- [ ]  Action Recognition
- [ ]  Video Summarization

cc. @DiwakarBasnet @sazio @sanae-a11y @jungnerd
bcc. @johko @lunarflu @merveenoyan

With our curriculum now finalized and our initial meeting behind us, we're eager to write the contents. Here are some things to keep in mind while writing:

Currently, we're leaning towards PyTorch as our primary library for the demos and internal scripts. https://github.com/johko/computer-vision-course/issues/26 and https://github.com/johko/computer-vision-course/issues/25.

**Comment by wo0jun** [comment]
Hi, @johko 👋
My teammates and I are working on this unit right now, however the issue of this unit is closed.
Should we open new issue about this unit?

**Comment by johko** [comment]
Hey @jungnerd , I closed all the chapter content issues that existed in the "Issues" section, but you can always feel free to open a new issue here in the "Discussion" section if it helps you to keep track/stay organized
